barretenberg/cpp/pil/vm2/range_check.pil

```diff
@@ -0,0 +1,189 @@
+namespace range_check(256);
+    // Range check selector
+    pol commit sel_rng_chk;
```

Comment:
We are now just using `sel` for the subtrace selector so that it looks like `range_check.sel` when the namespace is used.